### PR TITLE
feat(breakdowns): Add span_op_breakdowns fields to transactions and discover datasets.

### PR DESCRIPTION
**Pull request by dashed**
Depends on https://github.com/getsentry/snuba/pull/1762

Associated change in Relay: https://github.com/getsentry/relay/pull/934

Add `span_op_breakdowns` fields to the transaction and discover datasets so they can be read from Sentry.

## TODO

- [x] add tests for aggregate usage

### CODE REVIEW

File: tests/test_discover_api.py
Comment (evanh) [inline tests/test_discover_api.py:898]: Why is this here?

File: tests/test_discover_api.py
Comment (evanh) [inline tests/test_discover_api.py:869]: Having a test that closely mirrors how you intend to use this feature would also be nice. E.g. if you plan to aggregate/use them in conditions.

File: tests/test_discover_api.py
Comment (dashed) [reply tests/test_discover_api.py:898]: Copied it from existing test `test_individual_measurement`. Forgot to rename things to span_op_breakdowns

File: tests/test_discover_api.py
Comment (dashed) [inline tests/test_discover_api.py:1057]: This query was generated from a real Snuba query from my testing ingesting events (end to end) locally and producing aggregated/histogram values on the product side (i.e. frontend).

**Review by evanh — APPROVED** [review]
Please do not merge until the migration has been successfully run.

**Comment by dashed** [comment]
Will merge this to master after https://github.com/getsentry/snuba/pull/1762 is merged

**Pull request merged**
